### DISCUSSION TITLE
DOM-manipulation suggestion

### DISCUSSION THREAD

**Discussion by calebstem** [TheOdinProject/curriculum#22399] [Ideas]
For the continuation of the rock-papers-scissors project it recommends to "Copy your original code into a new file so you don’t lose it". This could be considered bad practice due to this project being on GitHub (or at least should be). This could be a learning opportunity to explain how branches work/checkout. I would do this, but I don't really believe myself to be qualified/understand exactly what the expectations for editing a lesson is.

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @calebstem, closing since we introduce branches in this lesson now https://www.theodinproject.com/lessons/foundations-revisiting-rock-paper-scissors#using-branches
